tests/unit/command/test_experiments.py

```diff
@@ -58,6 +58,32 @@ def test_experiments_diff(dvc, scm, mocker):
     )
 
 
+@pytest.mark.parametrize("show_json,show_csv", [(True, False), (False, True)])
+def test_experiments_show_format(dvc, scm, mocker, show_json, show_csv):
```

Comment:
We may need a functional test for this instead, as csv and the default output might have significant differences.